mobile/lib/constants/api.dart

```diff
@@ -30,6 +30,8 @@ class AirQoUrls {
   static String get locationHistory =>
       '${Config.airqoApi}/v2/users/locationHistory';
 
+  static String get searchAirQuality => '${Config.airqoApi}/v2/predict/search';
+
```

Comment:
Predict API

Reply:
Hi @NoahNsimbe , I keep on seeing error messages in my SMS regarding the predict search job. 

Are we really ready to introduce these relatively unstable solutions into our mobile app? cc: @BenjaminSsempala / @Mnoble-19 

Otherwise, I am open to a thorough review before making any "ML" solution as a default option in any of our solutions.

Reply:
Hello  @Baalmart 
Is the search API being used in the platform?